.github/reference-workflows/CI_1_1_1.yaml

```diff
@@ -59,7 +59,7 @@ jobs:
           pytest -v --cov=prj_1_1_1 --cov-report=xml --color=yes prj_1_1_1/tests/
 
       - name: CodeCov
-        uses: codecov/codecov-action@v1
+        uses: codecov/codecov-action@v5
         with:
           file: ./coverage.xml
```

Comment:
It looks like they have deprecated `file` in favor of `files`. See [codecov-action](https://github.com/codecov/codecov-action). We may want to update it to stay ahead.